src/libPMacc/include/memory/buffers/HostBufferIntern.hpp

```diff
@@ -55,7 +56,7 @@ class HostBufferIntern : public HostBuffer<TYPE, DIM>
     }
 
     HostBufferIntern(HostBufferIntern& source, DataSpace<DIM> dataSpace, DataSpace<DIM> offset=DataSpace<DIM>()) :
```

Comment:
`offset`: shouldn't this use a `create` function or is this constructor save to be inited with `0`s?